packages/cozy-harvest-lib/src/components/KonnectorModal.jsx

```diff
@@ -61,7 +61,7 @@ export class KonnectorModal extends PureComponent {
 
     try {
       const account = await findAccount(triggers.getAccountId(trigger))
-      this.setState({ account })
+      this.setState({ account, trigger })
```

Comment:
I think we should set the trigger in state whatever happen, to avoid being in the situation where no trigger is provided to TriggerManager and it behaves like for a trigger creation. This behaviour is not yet handled by this modal.

Reply:
If the accounts fail, it will be in the errored status, which is [correctly handled here](https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest-lib/src/components/KonnectorModal.jsx#L120)